plugins/lang/admincmd.txt

```diff
@@ -128,8 +128,8 @@ UNABLE_PAUSE = Server konnte das Spiel nicht anhalten. Reale Spieler werden auf
 SERVER_PROC = Server fuehrt %s aus
 PAUSING = Pausieren
 UNPAUSING = Pause beendet
-PAUSE = Pause
-UNPAUSE = Pause beendet
+PAUSE = pausiert den
+UNPAUSE = beendet die Pause auf dem
```

Comment:
I see what you're trying to do here, but we can't change it like this. This language key might be used by other plugins, so your variant is way too specific to the way admincmd uses it. If anything, it's admincmd that needs to be changed.

We could consider translating all the unpause variants with "fortsetzen".

Reply:
Yeah, I see that problem too, but did not find another solution... 
I'm running this for years that way and had no problems with other plugins - but I do have not that much plugins on my server.
It is used "[Name of Admin] PAUSE/UNPAUSE [Name of Server]" I think...